lua/super-commit/super-commit.lua

```diff
@@ -1,20 +1,24 @@
 local window_open = require('super-commit/window/open')
 local git_cmds = require('super-commit/git/commands')
+local map_table = git_cmds.map_table
 
 local M = {}
 
 function M.init()
   window_open.setup()
-  local bufnum_table = window_open.bufnum_table
-  local command_table = git_cmds.command_table
-  for k, cmd in pairs(command_table) do
-    git_cmds.show_output(bufnum_table[k], cmd)
-  end
+  git_cmds.show_output(map_table["git-status"].bufnum, map_table["git-status"].command)
+  git_cmds.show_output(map_table["filelist"].bufnum, map_table["filelist"].command)
+  git_cmds.show_output(map_table["git-diff-init"].bufnum, map_table["git-diff-init"].command)
```

Comment:
Improve readability by holding command keys in a set.

`init_commands` can be also used when setting winnum and bufnum, therefore it may be better to define this in `window/open.lua`. 

```suggestion
local init_commands = {"git-status", "filelist", "git-diff-init"}

-- or
-- local init_commands = window_open.init_commands
-- if defined in window/open.lua

function M.init()
  window_open.setup()
  for k in init_commands do
    wn, bn, cm = map_table[k]
    git_cmds.show_output(bn, cm)
```

Reply:
Thank you for providing code examples. 
(Same comment from me, please resolve this at the same time: https://github.com/supersaintex/super-commit.nvim/pull/5#discussion_r1809076684)

Reply:
Fixed it in 3189900d61664541177854465639614cf10efd0a